packages/react-components/src/link/src/useLink.ts

```diff
@@ -1,6 +1,32 @@
-import { cssModule, mergeClasses, normalizeSize, useAutoFocus, useMergedRefs } from "../../shared";
+import { ForwardedRef } from "react";
+import { MergedRef, Size, cssModule, mergeClasses, normalizeSize, useAutoFocus, useMergedRefs } from "../../shared";
 import { isNumber } from "lodash";
 
+export interface UseLinkProps {
+    cssModule?: string;
+    omitSize?: boolean;
+    color?: string;
+    underline?: string;
+    shape?: string;
+    external?: boolean;
+    autoFocus?: boolean | number;
+    size?: Size;
+    active?: boolean;
+    focus?: boolean;
+    hover?: boolean;
+    visited?: boolean;
+    target?: string;
+    rel?: string;
+    forwardedRef: ForwardedRef<HTMLElement>;
+}
+
+export interface UseLinkApi {
```

Comment:
If we need these, I suggest we use Chakra "Return" naming convention. Could also use "Output" instead of "Return"